downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc

```diff
@@ -50,8 +50,8 @@ When registration is complete, the registration page for the Automation Controll
 . In the *Automation controller application registration page* on Azure, copy and save the value of _Application (client) ID_.
 +
 You will use this value for the _Azure AD OAuth2 Key_ in the {PlatformNameShort} settings.
-. Under *Manage*, click *Certificates & secrets*.
-. Click *Client secrets* and then *+ New client secret*.
+. Under *Manage*, click btn:[Certificates & secrets].
```

Comment:
I believe these might be menu selections. Suggest rewriting to:

Select menu:Manage[Certificates & secrets].